### PR TITLE
Repairs bug and reimplements shorthand for tables into database query.

### PR DESCRIPTION
**Pull request by jayrbarr**
Revised pull request after repairing bug when style guide asked for change in destructuring the input request on the API endpoint.

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -93,7 +93,7 @@
...
... (typeof Reflect === \"undefined\" || !Reflect.construct) return false; if (Reflect.construct.sham) return false; if (typeof Proxy === \"function\") return true; try { Date.prototype.toString.call(Reflect.construct(Date, [], function () {})); return true; } catch (e) { return false; } }\n\nfunction _getPrototypeOf(o) { _getPrototypeOf = Object.setPrototypeOf ? Object.getPrototypeOf : function _getPrototypeOf(o) { return o.__proto__ || Object.getPrototypeOf(o); }; return _getPrototypeOf(o); }\n\nvar App = /*#__PURE__*/function (_React$Component) {\n  _inherits(App, _React$Component);\n\n  var _super = _createSuper(App);\n\n  function App(props) {\n    _classCallCheck(this, App);\n\n    return _super.call(this, props);\n  }\n\n  _createClass(App, [{\n    key: \"componentDidMount\",\n    value: function componentDidMount() {\n      fetch();\n    }\n  }, {\n    key: \"render\",\n    value: function render() {\n      return /*#__PURE__*/React.createElement(\"div\", null, \"Works!!!\");\n    }\n  }]);\n\n  return App;\n}(React.Component);\n\n;\nReactDOM.render( /*#__PURE__*/React.createElement(App, null), document.getElementById('app'));\n\n//# sourceURL=webpack:///./client/index.jsx?");
```
Comment (JasperC1999) [inline public/main.js:96]: What is this for?

File: server/index.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ app.use(express.static(path.join(__dirname, '../public')));
 
 app.listen(PORT, () => console.log(`Listening at http://localhost:${PORT}`));
 
-app.get('/:id', ({ id }, res) => {
+app.get('/:id', ({ params: { id } }, res) => {
```
Comment (JasperC1999) [inline server/index.js:15]: that's a clean way to do it!

Comment (jayrbarr) [reply server/index.js:15]: It's sort of what airbnb required - and it's what was messing me up for awhile, haha!

File: public/main.js (hunk 1 above)
Comment (jayrbarr) [reply public/main.js]: It's in that compiled main.js file, so I'm not exactly sure. I thought I had that gitignored.

**Pull request merged**